File-Formats/PDF/Working-with-OCR/net-core.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Perform OCR on a PDF document in ASP.NET Core | Syncfusion
```

Comment:
Change the content as like the comments in ASP.NET MVC

Reply:
Changed the content